### PR TITLE
chezmoi: update to v2.7.5

### PR DESCRIPTION
**Pull request by torculus**
Updating `chezmoi`, a utility to synchronize configuration files (aka "dotfiles")

This update also trims down my old patch file so that more metadata can successfully be included. It should make the component easier to maintain in the future

### CODE REVIEW

**Review by Toasterson — APPROVED** [review]
Nice work, I will have a testrun once I am out of the Plane again.  But I do not expect it to run into any trouble.

For the future patch, could we make an upstream issue about this? The Makefile is a bit to strongly bound to their CI to be useful for packaging in general. It should not assume to always be running in a git repo and not cross-build by default

**Comment by torculus** [comment]
@Toasterson, should we just wait on the next point release from upstream before merging this? That will probably incorporate the changes in the Makefile that make the patch unnecessary

**Comment by Toasterson** [comment]
We can also merge this and wait on the next release to remove the patch. I am for that.

**Review by Toasterson — APPROVED** [review]
LGTM

**Pull request merged**
